lib/conn-pool.js

```diff
@@ -63,7 +63,7 @@ export default class ConnPool {
 
         // Start uTP
         debug('creating utpServer in port %s', this.tcpServer.address().port)
-        this.utpServer.listen(this.tcpServer.address().port)
+        this.utpServer.listen(this.tcpServer.address().port, this._client.torrentHost === '::' ? '0.0.0.0' : this._client.torrentHost)
```

Comment:
Please add a comment stating why we're overriding `::` to `0.0.0.0`, so when `utp-native` finally supports v6, we know to remove it.